### PR TITLE
Doc/glossary

### LINKED ISSUE #534: Create separate page for glossary of terms
It is currently implemented as a tooltip.

### PR DESCRIPTION
**Pull request by mdbenito**
### Description

This PR adds a glossary page. I almost exclusively took terms from `abbreviations.md` but we should add more that are relevant to pydvl. Links to wikipedia / other sources might be in order too.

@schroedk the influence part is again a stub, please check it for consistency and completeness. @AnesBenmerzoug let me know of any terms that you miss and I'll try to add them.

At some point we will want to link to these from other docs. So maybe I'll try adding explicit anchors with `{ #anchor }`, although that has not really worked out so well in the past.

### Changes

- Closes #534 with a new glossary page. 
- Sorted the abbreviations page again.
- Some minor improvements to docs for the least core.

### Checklist

- [ ] ~Wrote Unit tests (if necessary)~
- [x] Updated Documentation (if necessary)
- [x] Updated Changelog
- [ ] ~If notebooks were added/changed, added boilerplate cells are tagged with `"tags": ["hide"]` or `"tags": ["hide-input"]`~

### CODE REVIEW

File: docs/value/the-core.md [hunk 1]
```diff
@@ -74,18 +79,20 @@ values = compute_least_core_values(utility, mode="exact")
 Because the number of subsets $S \subseteq D \setminus \{i\}$ is
 $2^{ | D | - 1 }$, one typically must resort to approximations.
 
-The simplest approximation consists in using a fraction of all subsets for the
-constraints. [@yan_if_2021] show that a quantity of order
-$\mathcal{O}((n - \log \Delta ) / \delta^2)$ is enough to obtain a so-called
-$\delta$-*approximate least core* with high probability. I.e. the following
-property holds with probability $1-\Delta$ over the choice of subsets:
+The simplest on consists in using a fraction of all subsets for the constraints.
```
Comment (AnesBenmerzoug) [inline docs/value/the-core.md:82]:
```suggestion
The simplest one consists in using a fraction of all subsets for the constraints.
```

**Comment by AnesBenmerzoug** [comment]
The new Glossary page makes the Methods page a bit redundant because it contains descriptions of some of the methods as well as links to the implementations. Perhaps we can keep method descriptions in Methods and only put other terms in the Glossary.

**Comment by mdbenito** [comment]
> The new Glossary page makes the Methods page a bit redundant because it contains descriptions of some of the methods as well as links to the implementations. Perhaps we can keep method descriptions in Methods and only put other terms in the Glossary.

Well... you suggested it 😅 But I'm not sure it's redundant. The methods page is a quick list with links for reference for newcomers ("is my algorithm here?"). Whereas the glossary is for quick definitions that can be linked from anywhere. I would imagine that in time, if we really want the glossary to be useful, we'd cross-ref to it from all over the docs, possibly adding a little tooltip with the definition (using e.g. the couple of js+css files suggested [here](https://github.com/ObsidianPublisher/tooltips-internal-link)).

**Comment by AnesBenmerzoug** [comment]
That sounds like a lot of work. I think we can merge it now as it is.

**Review by AnesBenmerzoug — APPROVED** [review]

**Pull request merged**